Source/DataTypes/SvgOrientConverter.cs

```diff
@@ -6,48 +6,40 @@ namespace Svg.DataTypes
 {
     public sealed class SvgOrientConverter : TypeConverter
     {
-        public override object ConvertFrom(ITypeDescriptorContext context, System.Globalization.CultureInfo culture, object value)
+        public override object ConvertFrom(ITypeDescriptorContext context, CultureInfo culture, object value)
         {
             if (value == null)
-            {
-                return new SvgUnit(SvgUnitType.User, 0.0f);
```

Comment:
I do not know why it was `SvgUnit`.
Surely I think that this condition did not become true.

Reply:
Checking the history, it looks like this got mixed in during an earlier revert a few years ago - obviously a mistake.